client-src/elements/chromedash-app.js

```diff
@@ -113,6 +113,7 @@ class ChromedashApp extends LitElement {
     page('/newfeatures', (ctx) => {
       this.pageComponent = document.createElement('chromedash-all-features-page');
       this.pageComponent.user = this.user;
+      this.pageComponent.rawQuery = window.csClient.parseRawQuery(ctx.querystring);
```

Comment:
I'm happy to see that this part worked out pretty cleanly.   I am curious to see if this code is executed when you change the URL (as part of your upcoming PR).